### PR TITLE
ci: Fetch Dataplan on Release Build

### PR DESCRIPTION
**Pull request by alexs-mparticle**
## Summary
- Adding an action that will fetch the current data plan and commit it to the repo within a `dataplans` folder within the Higgs Shop Sample app.
- This will run during the Build and Release process for now, but I would anticipate moving this to every PR build when we are able to integrate the linter or validation.

## Master Issue
- Closes https://go.mparticle.com/work/SQDSDKS-3682

### CODE REVIEW

File: .github/workflows/higgs-shop-sample-app-release.yml [hunk 1]
```diff
@@ -31,11 +31,65 @@ jobs:
...
+            run: pwd
+            
+          - name: View directory
+            run: ls -altr .
+            
+          - name: View directory
+            run: ls -altr dataplans
+            
+          - name: View Github Workspace
+            run: ls -altr ${{ github.workspace }}
```
Comment (rmi22186) [inline .github/workflows/higgs-shop-sample-app-release.yml:70]: Is this for debugging purposes when you were building this script - if so should they be removed?  if not, i think you should change the `name` for each to more accurately reflect what it's doing

1. Print directory path
2. View all files 
3. View all files that start with `dataplans`

**Review by rmi22186 — CHANGES_REQUESTED** [review]
a few comments, but I think maybe @mchuangatmp should take a look since he has a better understanding of the sample app ci cd than i do.

File: .github/workflows/higgs-shop-sample-app-release.yml (hunk 1 above)
Comment (alexs-mparticle) [reply .github/workflows/higgs-shop-sample-app-release.yml]: Good catch. Removing them.

**Comment by rmi22186** [comment]
LGTM

**Review by rmi22186 — APPROVED** [review]

**Pull request merged**